test/controllers/organization_membership_controller_test.exs

```diff
@@ -1,112 +1,70 @@
 defmodule CodeCorps.OrganizationMembershipControllerTest do
-  use CodeCorps.ApiCase
-
-  alias CodeCorps.OrganizationMembership
-  alias CodeCorps.Organization
-  alias CodeCorps.User
+  use CodeCorps.ApiCase, resource_name: :organization_membership
 
   @valid_attrs %{role: "contributor"}
-  @invalid_attrs %{role: "invalid_role"}
-
-  defp build_payload, do: %{ "data" => %{"type" => "organization-membership"}}
-  defp put_relationships_with_member(payload, organization, member) do
-    relationships = build_relationships(organization, member)
-    payload |> put_in(["data", "relationships"], relationships)
-  end
-
-  defp build_relationships(organization, member) do
-    %{
-      organization: %{data: %{id: organization.id}},
-      member: %{data: %{id: member.id}}
-    }
-  end
-
-  defp assert_role(data, role) do
-    assert data["attributes"]["role"] == role
-    data
-  end
 
   describe "index" do
     test "lists all resources", %{conn: conn} do
       [membership_1, membership_2] = insert_pair(:organization_membership)
 
-      path = conn |> organization_membership_path(:index)
-      response = conn |> get(path) |> json_response(200)
-
-      assert ids_from_response(response) == [membership_1.id, membership_2.id]
+      conn
+      |> request_index
+      |> json_response(200)
+      |> assert_ids_from_response([membership_1.id, membership_2.id])
     end
 
     test "filters resources by membership id", %{conn: conn} do
       [membership_1, membership_2] = insert_pair(:organization_membership)
       insert(:organization_membership)
```

Comment:
Similarly to a comment in my other review, while this is good, it could be rewritten in one line as 

```Elixir
[membership_1, membership_2 | _] = insert_list(3, :organization_membership)
```